### PR TITLE
show sidebar in campus edit form

### LINKED ISSUE #186: add the sidebar to the campus form

### PR DESCRIPTION
**Pull request by boney-bun**
fix #186

### CODE REVIEW

**Comment by boney-bun** [comment]
PIA:
ford3#186

**Review by Christiaanvdm — DISMISSED** [review]
LGTM

**Review by Christiaanvdm — COMMENTED** [review]
I believe this will show the side-bar on the provider form as well which is unwanted.
I believe just adding this to the campus' javascript should be sufficient.

```
$(document).ready(function() {
    // Show sidebar
    toggleSidebar()
})

```

**Comment by pierrealixt** [comment]
Why the name of the provider disappears?

**Comment by boney-bun** [comment]
@Christiaanvdm i'm a bit confusing about the sidebar part.
the original codes seem to not working properly.
even in the test_widgets it's not working.

so i change the code in 8308d5f to using jquery instead of using HTML DOM property.

below is how the sidebar behave:
ford3#208

notice that the sidebar always show up on detail pages, but hide on edit pages.
if this is the expected behaviour, then this issue is irrelevant.
or, do we want to make an exception to show the sidebar in the campus edit form ?

File: django_project/core/base_static/css/sidebar.css [hunk 1]
```diff
@@ -10,7 +10,6 @@
     background-color: var(--c-purple);
     color: #fff;
     transition: all 0.3s;
-    margin-left: -250px;
```
Comment (Christiaanvdm) [inline django_project/core/base_static/css/sidebar.css]: You need to undo this change for the sidebar.toggle() to work correctly.

File: django_project/core/base_static/js/sidebar.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
 const toggleSidebar = () => {
```
Comment (Christiaanvdm) [inline django_project/core/base_static/js/sidebar.js]: I think if the CSS is reverted the JS doesnt need to be changed.

**Comment by boney-bun** [comment]
hi @Christiaanvdm 
i think i'm a bit confused with the logic before.
i have returned the code into the original one and add campus.js into campus edit form.

**Review by pierrealixt — CHANGES_REQUESTED** [review]
sidebar-campus-form
The provider's name is not present in the sidebar...

**Comment by boney-bun** [comment]
@pierrealixt the provider link is now show up in campus edit form:
ford3#208revise

**Review by pierrealixt — APPROVED** [review]
LGTM

**Comment by boney-bun** [comment]
thanks @pierrealixt 
i'm going to merge this

**Pull request merged**